src/tcgwars/logic/impl/gen8/BattleStyles.groovy

```diff
@@ -384,6 +378,7 @@ public enum BattleStyles implements LogicCardInfo {
           text "Once during your turn, you may draw cards until you have 3 cards in your hand. If this Pokémon is in the Active Spot, you may draw cards until you have 4 cards in your hand instead. You can't use more than 1 Exciting Stage Ability each turn."
           actionA {
             checkLastTurn()
+            assert bg.em().retrieveObject("$self $thisAbility") != bg.turnCount : "Only 1 $thisAbility can be used per turn."
```

Comment:
I don't think this works. Since you included the pcs in the key, wouldn't another ability from a different pcs be able to be used on the same turn?

Reply:
In this case, $self is the string "Krikitune V (BST 6)" which is the same for all Krikitune V right now, but that would be an issue for a reprint yeah, I'll fix that.